lib/cn/node_split.c

```diff
@@ -484,9 +487,14 @@ kvset_split_res_init(struct cn_compaction_work *w, struct kvset_split_res *resul
 static void
 kvset_split_res_free(struct kvset *ks, struct kvset_split_res *result)
 {
+    if (!ks)
```

Comment:
Just return early if the second argument is NULL too?

Reply:
The second argument is contained within the work args struct, so can't be NULL.